libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Security/SFUserAccountManager.m

```diff
@@ -392,20 +396,48 @@ - (BOOL)deleteAccountForUser:(SFUserAccount *)user error:(NSError **)error {
 }
 
 - (SFUserAccount *)applyCredentials:(SFOAuthCredentials*)credentials {
+    return [self applyCredentials:credentials withIdData:nil];
+}
+
+- (SFUserAccount *)applyCredentials:(SFOAuthCredentials*)credentials withIdData:(SFIdentityData *) identityData {
+    return [self applyCredentials:credentials withIdData:identityData andNotification:YES];
+}
 
-    SFUserAccountChange change = SFUserAccountChangeCredentials;
+- (SFUserAccount *)applyCredentials:(SFOAuthCredentials*)credentials withIdData:(SFIdentityData *) identityData andNotification:(BOOL) shouldSendNotification{
```

Comment:
@khawkins Can be used by  anyone consuming this API directly and not wanting notifications but simply addition of the user account.